solr/test-framework/src/java/org/apache/solr/SolrTestCase.java

```diff
@@ -124,6 +126,17 @@
 
   protected volatile static PerThreadExecService testExecutor;
 
+  private static final CryptoKeys.RSAKeyPair reusedKeys = new CryptoKeys.RSAKeyPair();
```

Comment:
Why not use the key-pair that's already on disk? See https://github.com/apache/lucene-solr/blob/master/solr/test-framework/src/java/org/apache/solr/SolrTestCaseJ4.java#L289-L290

Reply:
This is for just testcase